src/components/ec-table/ec-table.story.js

```diff
@@ -95,116 +98,55 @@ stories
         default: text('maxHeight', '400px'),
       },
       stickyColumn: {
-        default: select('stickyColumn', ['left', 'right'], 'left'),
+        default: select('stickyColumn', { none: undefined, left: 'left', right: 'right' }, undefined),
       },
     },
     methods: {
       onSort: action('sort'),
       onRowClick: action('onRowClick'),
     },
     template: `
-      <div style="display: flex; height: 100vh">
-        <div style="margin: auto 20px; width: 90vw" class="ec-card" >
+      <div class="tw-flex tw-flex-col tw-h-screen">
+        <h2 class="feature-title tw-m-24">Basic</h2>
+        <div class="tw-my-auto tw-mx-20 ec-card" style="width: 90vw">
```

Comment:
any reason why keeping the width and not making it 100%? Can't remember why

Reply:
I am not sure...I tried with 100% and the tables overflow the viewport (at least in my laptop)

Reply:
This because if you set to 100% it will break the table on the story